AdventureCapitalist/App.js

```diff
@@ -1,23 +1,60 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState } from 'react';
+import { StyleSheet, View, Button } from 'react-native';
 import GameScreen from './src/screens/GameScreen';
-import config from './config/development'
+import config from './config/base';
```

Comment:
change to production
```suggestion
import config from './config/production';
```